remix/app/components/AppLink/index.tsx

```diff
@@ -1,15 +1,20 @@
 import { Link } from '@remix-run/react';
 
-interface ButtonProps {
+import { setButtonType } from './helper';
+
+interface AppLinkProps {
   name: string;
   href: string;
+  color?: string;
 }
 
-export default function AppLink({ name, href }: ButtonProps) {
+export default function AppLink({ name, href, color }: AppLinkProps) {
```

Comment:
Ditty here on switching to `type`

Reply:
Updated here: [9b0ebe5](https://github.com/nimblehq/js-framework-benchmark/pull/130/commits/9b0ebe5ec2af4123c6639502977bb5a584cbc878)